packages/react/src/components/DataTable/TableToolbar.js

```diff
@@ -5,32 +5,45 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import cx from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { settings } from 'carbon-components';
 import { AriaLabelPropType } from '../../prop-types/AriaPropTypes';
 
 const { prefix } = settings;
 
-const TableToolbar = ({ children, ...rest }) => (
-  <section {...rest} className={`${prefix}--table-toolbar`}>
-    {children}
-  </section>
-);
+const TableToolbar = ({ children, size, ...rest }) => {
```

Comment:
`size` seems to always be equal to `normal`. Do we need to update anything in the storybook examples to get this to work properly?